Add logs section (#3097)
DataDog/integrations-core: postfix/README.md

[Restart the Agent][5] to start sending Postfix metrics to Datadog.

#### Log Collection

**Available for Agent >6.0**

> Postfix logs activity to the syslog daemon that then gets sent to logfiles.

Reviewer: ```suggestion
Postfix sends logs to the syslog daemon, which then writes logs to the file system.
```

[Restart the Agent][5] to start sending Postfix metrics to Datadog.

#### Log Collection

**Available for Agent >6.0**

Postfix logs activity to the syslog daemon that then gets sent to logfiles.

> By default, the naming convention and log files can by configured in your syslog file:

Reviewer: ```suggestion
The naming convention and log file destinations are configurable:
```

```
/etc/syslog.conf:
    mail.err                                    /dev/console
    mail.debug                                  /var/log/maillog
```

> * Collecting logs is disabled by default in the Datadog Agent, enable it in your `datadog.yaml` file with:

Reviewer: ```suggestion
* Collecting logs is disabled by default in the Datadog Agent. Enable it in your `datadog.yaml` file:
```

Author: This is present in a few other README's as well, as I took it from there. But I can change it to this :)

Reviewer 2: The double sentences one is better since it's more "business". Simple sentence for simple concepts.
